src/Components/Artwork/Details.tsx

```diff
@@ -89,14 +91,70 @@ export class Details extends React.Component<Props, null> {
     }
   }
 
+  saleInfoLine() {
+    const { artwork } = this.props
+    const { sale } = artwork
+    const inClosedAuction = sale && sale.is_auction && sale.is_closed
+
+    return (
+      <>
+        <Sans
+          style={{ display: "inline" }}
+          color={color("black100")}
+          weight={"medium"}
+          size={"2"}
+        >
+          {inClosedAuction ? "Bidding closed" : this.saleMessageOrBidInfo()}{" "}
+        </Sans>
+        <Sans style={{ display: "inline" }} size={"2"} color={color("black60")}>
+          {!inClosedAuction && this.auctionInfo()}
+        </Sans>
+        <Spacer mb={0.3} />
+      </>
```

Comment:
This might be worth a follow up, but its certainly not hurting anything :) 

```jsx

```
<Flex mb={0.3}>
  <Sans size='2' color='black100' weight='medium'>...</Sans>
  <Sans size='2' color='black60'>...</Sans>
</Flex>
```

Flex by default displays items inline (unless you change it to `flexDirection='column'` and don't need the color function in this case (we fixed that issue a little while back).

Reply:
Oh dope! Yea this looks way better, good call on flexbox.